src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsPlatformImage.ps1

```diff
@@ -123,39 +123,10 @@ function Get-AzsPlatformImage {
             $offer = $ArmResourceIdParameterValues['offer']
             $sku = $ArmResourceIdParameterValues['sku']
             $version = $ArmResourceIdParameterValues['version']
-        } elseif ( -not $PSBoundParameters.ContainsKey('Location')) {
+        } elseif ( [System.String]::IsNullOrEmpty($Location)) {
             $Location = (Get-AzureRMLocation).Location
         }
 
-        $filterInfos = @(
```

Comment:
Why this is being removed?

Reply:
@cheguv We have to leave this in, I forgot that this was for client side filters.

Reply:
added back in.

Reply:
I still see it as removed.